src/app/(sidebar)/account/saved/page.tsx

```diff
@@ -0,0 +1,238 @@
+"use client";
+
+import { useCallback, useEffect, useState } from "react";
+import { Alert, Text, Card, Input, Icon, Button } from "@stellar/design-system";
+
+import { Box } from "@/components/layout/Box";
+import { InputSideElement } from "@/components/InputSideElement";
+import { SaveKeypairModal } from "@/components/SaveKeypairModal";
+import { SavedItemTimestampAndDelete } from "@/components/SavedItemTimestampAndDelete";
+
+import { useStore } from "@/store/useStore";
+import { localStorageSavedKeypairs } from "@/helpers/localStorageSavedKeypairs";
+import { NetworkOptions } from "@/constants/settings";
+import { useIsTestingNetwork } from "@/hooks/useIsTestingNetwork";
+
+import { NetworkType, SavedKeypair } from "@/types/types";
+import { arrayItem } from "@/helpers/arrayItem";
+
+export default function SavedKeypairs() {
```

Comment:
curious - when should we use `export default function` vs. `export const`. in vibrant days, we preferred using `const` for exporting vs. default ([product convention](https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#all-files-should-only-export-named-variables-not-defaults)).

Reply:
(I am not suggesting that we follow the convention btw, I'm just curious)

Reply:
In this case ([page.tsx](https://nextjs.org/docs/14/app/api-reference/file-conventions/page)), we need to follow the Next.js convention and any other framework level file ([examples](https://nextjs.org/docs/14/app/api-reference/file-conventions)). I agree that we should use named exports when possible. :+1: